docs/pages/dropdown.js

```diff
@@ -625,77 +546,23 @@ function TruncationDropdownExample() {
         </MainSection.Subsection>
       </MainSection>
       <MainSection name="Subcomponents" />
-      <PropTable
+      <GeneratedPropTable
```

Comment:
I'm still seeing `index` in the tables for Item and Link…looks like that needs to be added to `excludeProps` here

Reply:
DONE